pkg/api/org.go

```diff
@@ -152,6 +152,11 @@ func updateOrgAddressHelper(form dtos.UpdateOrgAddressForm, orgID int64) respons
 
 // GET /api/orgs/:orgId
 func DeleteOrgByID(c *models.ReqContext) response.Response {
+	// before deleting an org, check if user does not belong to the current org
+	if c.OrgId == c.ParamsInt64(":orgId") {
+		return response.Error(403, "Can not delete org for current user", nil)
```

Comment:
I think 400 makes more sense. 403 implies that the user don't have permission to access this resource. WDYT?

```suggestion
		return response.Error(400, "Can not delete org for current user", nil)
```